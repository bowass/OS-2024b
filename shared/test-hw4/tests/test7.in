58781
3216
58451
30074
35315
29710
38834
37062
975
2062
43285
41305
14236
12258
56802
2536
26191
41983
60807
40248
53361
60014
45195
39209
36796
1271
9517
14631
29183
51792
28288
17602
33492
61826
2577
10414
24680
13766
8697
9578
29869
30485
30558
18303
30213
3214
47626
54366
30222
42898
54075
36633
30888
23295
24753
8768
48639
16703
37095
54092
7862
57133
47612
17425
20809
32466
6326
58521
35163
2699
28
50883
2458
47929
60885
549
26172
618
6925
21288
35666
30668
7201
12087
24770
47001
57564
49495
19679
50737
1508
32880
45854
15080
8939
62896
37780
14159
5158
62297
29585
36302
51016
34002
57923
49513
65110
7089
2364
55982
41461
59849
3341
57600
22487
19470
13473
50442
20570
33733
55617
43741
13142
320
27228
59642
56488
65379
26182
52119
29171
25625
47130
48539
4648
55475
36468
32697
20409
2015
52320
14992
33194
20208
62800
55599
31401
27038
58717
25220
33114
50270
14688
59328
28930
64485
15847
46814
23839
14305
16356
10103
6290
61725
26863
41964
35617
27147
63346
27503
5600
17564
37914
65411
21944
51505
25691
42381
39216
21790
47990
26863
64230
37535
38215
41571
32915
31081
53654
41680
22441
52709
12966
55052
42964
29493
6609
40324
50135
29772
53931
22330
44173
29943
36807
4257
32812
1860
764
48284
17608
48830
35197
24180
52603
52717
53734
19923
12117
55930
32701
53840
24436
7139
63496
56643
50887
49356
57212
55174
26352
30830
37682
20496
13942
20418
45803
3435
37008
5706
49207
48964
57150
11235
64866
31647
42633
26669
37568
43255
46429
12458
16702
22106
31742
28830
47512
56206
63012
2967
22442
33520
41328
56244
46288
31239
35798
50808
237
228
60113
22359
5218
25228
49927
52890
20556
25730
11148
43553
57667
42375
13499
35199
32731
34348
1109
24950
43
18957
34178
11420
1507
46048
20764
37373
23408
23650
44584
26526
57757
39542
46058
22916
59969
36211
45028
65476
41403
60213
27216
43341
7839
58529
35358
52859
25332
33629
16983
42033
14874
2756
20496
15854
55905
35239
63124
13058
38369
51661
55762
44955
19703
40476
14617
14541
10194
7742
50663
538
41269
24184
1432
26682
46321
42449
41165
63451
4409
41396
19012
12948
45420
34489
58291
1390
46344
44121
43653
25089
15276
22231
31678
11061
14466
23931
6901
20175
14115
19388
62174
10836
41814
64152
13740
40925
25980
57229
45733
55823
60387
40803
40167
5366
1152
56817
48395
27299
14857
23137
37444
10549
56739
4277
63512
32464
7543
37542
28221
22095
49941
39006
863
26726
41044
2645
8972
17239
19364
48391
21824
22134
56368
41118
9373
49918
63423
17198
23095
23541
51393
34904
31152
56389
62874
23834
2407
27118
35531
27570
13545
1927
26545
38435
10597
35902
61972
24089
45655
27991
53533
49276
65321
49048
14480
59538
26633
64014
24132
29744
48966
55060
32906
1007
47546
23005
56364
59221
29842
30016
49466
56450
36802
22078
40292
57790
51417
1524
2566
1055
39881
53559
59040
31997
36733
46402
46774
41068
25303
42140
10818
30924
63933
21005
32369
52916
63207
28612
1934
39577
44227
25342
17622
20234
8961
56686
40834
44721
43310
1604