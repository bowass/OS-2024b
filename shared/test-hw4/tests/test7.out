Virtual address: 58781 Physical address: 157 Value: 0
Virtual address: 3216 Physical address: 400 Value: 0
Virtual address: 58451 Physical address: 595 Value: 20
Virtual address: 30074 Physical address: 890 Value: 29
Virtual address: 35315 Physical address: 1267 Value: 124
Virtual address: 29710 Physical address: 1294 Value: 29
Virtual address: 38834 Physical address: 1714 Value: 37
Virtual address: 37062 Physical address: 1990 Value: 36
Virtual address: 975 Physical address: 2255 Value: -13
Virtual address: 2062 Physical address: 2318 Value: 2
Virtual address: 43285 Physical address: 2581 Value: 0
Virtual address: 41305 Physical address: 2905 Value: 0
Virtual address: 14236 Physical address: 3228 Value: 0
Virtual address: 12258 Physical address: 3554 Value: 11
Virtual address: 56802 Physical address: 3810 Value: 55
Virtual address: 2536 Physical address: 4072 Value: 0
Virtual address: 26191 Physical address: 4175 Value: -109
Virtual address: 41983 Physical address: 4607 Value: -1
Virtual address: 60807 Physical address: 4743 Value: 97
Virtual address: 40248 Physical address: 4920 Value: 0
Virtual address: 53361 Physical address: 5233 Value: 0
Virtual address: 60014 Physical address: 5486 Value: 58
Virtual address: 45195 Physical address: 5771 Value: 34
Virtual address: 39209 Physical address: 5929 Value: 0
Virtual address: 36796 Physical address: 6332 Value: 0
Virtual address: 1271 Physical address: 6647 Value: 61
Virtual address: 9517 Physical address: 6701 Value: 0
Virtual address: 14631 Physical address: 6951 Value: 73
Virtual address: 29183 Physical address: 7423 Value: 127
Virtual address: 51792 Physical address: 7504 Value: 0
Virtual address: 28288 Physical address: 7808 Value: 0
Virtual address: 17602 Physical address: 8130 Value: 17
Virtual address: 33492 Physical address: 8404 Value: 0
Virtual address: 61826 Physical address: 8578 Value: 60
Virtual address: 2577 Physical address: 8721 Value: 0
Virtual address: 10414 Physical address: 9134 Value: 10
Virtual address: 24680 Physical address: 9320 Value: 0
Virtual address: 13766 Physical address: 9670 Value: 13
Virtual address: 8697 Physical address: 9977 Value: 0
Virtual address: 9578 Physical address: 6762 Value: 9
Virtual address: 29869 Physical address: 1453 Value: 0
Virtual address: 30485 Physical address: 10005 Value: 0
Virtual address: 30558 Physical address: 10078 Value: 29
Virtual address: 18303 Physical address: 10367 Value: -33
Virtual address: 30213 Physical address: 10501 Value: 0
Virtual address: 3214 Physical address: 398 Value: 3
Virtual address: 47626 Physical address: 10762 Value: 46
Virtual address: 54366 Physical address: 11102 Value: 53
Virtual address: 30222 Physical address: 10510 Value: 29
Virtual address: 42898 Physical address: 11410 Value: 41
Virtual address: 54075 Physical address: 11579 Value: -50
Virtual address: 36633 Physical address: 6169 Value: 0
Virtual address: 30888 Physical address: 11944 Value: 0
Virtual address: 23295 Physical address: 12287 Value: -65
Virtual address: 24753 Physical address: 9393 Value: 0
Virtual address: 8768 Physical address: 12352 Value: 0
Virtual address: 48639 Physical address: 12799 Value: 127
Virtual address: 16703 Physical address: 12863 Value: 79
Virtual address: 37095 Physical address: 2023 Value: 57
Virtual address: 54092 Physical address: 11596 Value: 0
Virtual address: 7862 Physical address: 13238 Value: 7
Virtual address: 57133 Physical address: 13357 Value: 0
Virtual address: 47612 Physical address: 13820 Value: 0
Virtual address: 17425 Physical address: 7953 Value: 0
Virtual address: 20809 Physical address: 13897 Value: 0
Virtual address: 32466 Physical address: 14290 Value: 31
Virtual address: 6326 Physical address: 14518 Value: 6
Virtual address: 58521 Physical address: 665 Value: 0
Virtual address: 35163 Physical address: 1115 Value: 86
Virtual address: 2699 Physical address: 8843 Value: -94
Virtual address: 28 Physical address: 14620 Value: 0
Virtual address: 50883 Physical address: 15043 Value: -80
Virtual address: 2458 Physical address: 3994 Value: 2
Virtual address: 47929 Physical address: 15161 Value: 0
Virtual address: 60885 Physical address: 4821 Value: 0
Virtual address: 549 Physical address: 15397 Value: 0
Virtual address: 26172 Physical address: 4156 Value: 0
Virtual address: 618 Physical address: 15466 Value: 0
Virtual address: 6925 Physical address: 15629 Value: 0
Virtual address: 21288 Physical address: 15912 Value: 0
Virtual address: 35666 Physical address: 16210 Value: 34
Virtual address: 30668 Physical address: 10188 Value: 0
Virtual address: 7201 Physical address: 16417 Value: 0
Virtual address: 12087 Physical address: 3383 Value: -51
Virtual address: 24770 Physical address: 9410 Value: 24
Virtual address: 47001 Physical address: 16793 Value: 0
Virtual address: 57564 Physical address: 17116 Value: 0
Virtual address: 49495 Physical address: 17239 Value: 85
Virtual address: 19679 Physical address: 17631 Value: 55
Virtual address: 50737 Physical address: 14897 Value: 0
Virtual address: 1508 Physical address: 17892 Value: 0
Virtual address: 32880 Physical address: 18032 Value: 0
Virtual address: 45854 Physical address: 18206 Value: 44
Virtual address: 15080 Physical address: 18664 Value: 0
Virtual address: 8939 Physical address: 12523 Value: -70
Virtual address: 62896 Physical address: 18864 Value: 0
Virtual address: 37780 Physical address: 19092 Value: 0
Virtual address: 14159 Physical address: 3151 Value: -45
Virtual address: 5158 Physical address: 19238 Value: 5
Virtual address: 62297 Physical address: 19545 Value: 0
Virtual address: 29585 Physical address: 19857 Value: 0
Virtual address: 36302 Physical address: 20174 Value: 35
Virtual address: 51016 Physical address: 20296 Value: 0
Virtual address: 34002 Physical address: 20690 Value: 33
Virtual address: 57923 Physical address: 20803 Value: -112
Virtual address: 49513 Physical address: 17257 Value: 0
Virtual address: 65110 Physical address: 21078 Value: 63
Virtual address: 7089 Physical address: 15793 Value: 0
Virtual address: 2364 Physical address: 3900 Value: 0
Virtual address: 55982 Physical address: 21422 Value: 54
Virtual address: 41461 Physical address: 3061 Value: 0
Virtual address: 59849 Physical address: 21705 Value: 0
Virtual address: 3341 Physical address: 21773 Value: 0
Virtual address: 57600 Physical address: 22016 Value: 0
Virtual address: 22487 Physical address: 22487 Value: -11
Virtual address: 19470 Physical address: 17422 Value: 19
Virtual address: 13473 Physical address: 22689 Value: 0
Virtual address: 50442 Physical address: 22794 Value: 49
Virtual address: 20570 Physical address: 23130 Value: 20
Virtual address: 33733 Physical address: 23493 Value: 0
Virtual address: 55617 Physical address: 23617 Value: 0
Virtual address: 43741 Physical address: 24029 Value: 0
Virtual address: 13142 Physical address: 24150 Value: 12
Virtual address: 320 Physical address: 24384 Value: 0
Virtual address: 27228 Physical address: 24668 Value: 0
Virtual address: 59642 Physical address: 25082 Value: 58
Virtual address: 56488 Physical address: 25256 Value: 0
Virtual address: 65379 Physical address: 25443 Value: -40
Virtual address: 26182 Physical address: 4166 Value: 25
Virtual address: 52119 Physical address: 25751 Value: -27
Virtual address: 29171 Physical address: 7411 Value: 124
Virtual address: 25625 Physical address: 25881 Value: 0
Virtual address: 47130 Physical address: 26138 Value: 46
Virtual address: 48539 Physical address: 12699 Value: 102
Virtual address: 4648 Physical address: 26408 Value: 0
Virtual address: 55475 Physical address: 26803 Value: 44
Virtual address: 36468 Physical address: 26996 Value: 0
Virtual address: 32697 Physical address: 27321 Value: 0
Virtual address: 20409 Physical address: 27577 Value: 0
Virtual address: 2015 Physical address: 27871 Value: -9
Virtual address: 52320 Physical address: 28000 Value: 0
Virtual address: 14992 Physical address: 18576 Value: 0
Virtual address: 33194 Physical address: 28330 Value: 32
Virtual address: 20208 Physical address: 28656 Value: 0
Virtual address: 62800 Physical address: 18768 Value: 0
Virtual address: 55599 Physical address: 23599 Value: 75
Virtual address: 31401 Physical address: 28841 Value: 0
Virtual address: 27038 Physical address: 29086 Value: 26
Virtual address: 58717 Physical address: 93 Value: 0
Virtual address: 25220 Physical address: 29316 Value: 0
Virtual address: 33114 Physical address: 28250 Value: 32
Virtual address: 50270 Physical address: 29534 Value: 49
Virtual address: 14688 Physical address: 7008 Value: 0
Virtual address: 59328 Physical address: 29888 Value: 0
Virtual address: 28930 Physical address: 7170 Value: 28
Virtual address: 64485 Physical address: 30181 Value: 0
Virtual address: 15847 Physical address: 30439 Value: 121
Virtual address: 46814 Physical address: 30686 Value: 45
Virtual address: 23839 Physical address: 30751 Value: 71
Virtual address: 14305 Physical address: 3297 Value: 0
Virtual address: 16356 Physical address: 31204 Value: 0
Virtual address: 10103 Physical address: 31351 Value: -35
Virtual address: 6290 Physical address: 14482 Value: 6
Virtual address: 61725 Physical address: 8477 Value: 0
Virtual address: 26863 Physical address: 31727 Value: 59
Virtual address: 41964 Physical address: 4588 Value: 0
Virtual address: 35617 Physical address: 16161 Value: 0
Virtual address: 27147 Physical address: 24587 Value: -126
Virtual address: 63346 Physical address: 31858 Value: 61
Virtual address: 27503 Physical address: 32111 Value: -37
Virtual address: 5600 Physical address: 32480 Value: 0
Virtual address: 17564 Physical address: 8092 Value: 0
Virtual address: 37914 Physical address: 32538 Value: 37
Virtual address: 65411 Physical address: 25475 Value: -32
Virtual address: 21944 Physical address: 32952 Value: 0
Virtual address: 51505 Physical address: 33073 Value: 0
Virtual address: 25691 Physical address: 25947 Value: 22
Virtual address: 42381 Physical address: 33421 Value: 0
Virtual address: 39216 Physical address: 5936 Value: 0
Virtual address: 21790 Physical address: 32798 Value: 21
Virtual address: 47990 Physical address: 15222 Value: 46
Virtual address: 26863 Physical address: 31727 Value: 59
Virtual address: 64230 Physical address: 33766 Value: 62
Virtual address: 37535 Physical address: 33951 Value: -89
Virtual address: 38215 Physical address: 34119 Value: 81
Virtual address: 41571 Physical address: 34403 Value: -104
Virtual address: 32915 Physical address: 18067 Value: 36
Virtual address: 31081 Physical address: 34665 Value: 0
Virtual address: 53654 Physical address: 34966 Value: 52
Virtual address: 41680 Physical address: 34512 Value: 0
Virtual address: 22441 Physical address: 22441 Value: 0
Virtual address: 52709 Physical address: 35301 Value: 0
Virtual address: 12966 Physical address: 35494 Value: 12
Virtual address: 55052 Physical address: 35596 Value: 0
Virtual address: 42964 Physical address: 11476 Value: 0
Virtual address: 29493 Physical address: 19765 Value: 0
Virtual address: 6609 Physical address: 36049 Value: 0
Virtual address: 40324 Physical address: 4996 Value: 0
Virtual address: 50135 Physical address: 36311 Value: -11
Virtual address: 29772 Physical address: 1356 Value: 0
Virtual address: 53931 Physical address: 36523 Value: -86
Virtual address: 22330 Physical address: 22330 Value: 21
Virtual address: 44173 Physical address: 36749 Value: 0
Virtual address: 29943 Physical address: 1527 Value: 61
Virtual address: 36807 Physical address: 6343 Value: -15
Virtual address: 4257 Physical address: 37025 Value: 0
Virtual address: 32812 Physical address: 17964 Value: 0
Virtual address: 1860 Physical address: 27716 Value: 0
Virtual address: 764 Physical address: 15612 Value: 0
Virtual address: 48284 Physical address: 37276 Value: 0
Virtual address: 17608 Physical address: 8136 Value: 0
Virtual address: 48830 Physical address: 37566 Value: 47
Virtual address: 35197 Physical address: 1149 Value: 0
Virtual address: 24180 Physical address: 37748 Value: 0
Virtual address: 52603 Physical address: 35195 Value: 94
Virtual address: 52717 Physical address: 35309 Value: 0
Virtual address: 53734 Physical address: 35046 Value: 52
Virtual address: 19923 Physical address: 38099 Value: 116
Virtual address: 12117 Physical address: 3413 Value: 0
Virtual address: 55930 Physical address: 21370 Value: 54
Virtual address: 32701 Physical address: 27325 Value: 0
Virtual address: 53840 Physical address: 36432 Value: 0
Virtual address: 24436 Physical address: 38260 Value: 0
Virtual address: 7139 Physical address: 15843 Value: -8
Virtual address: 63496 Physical address: 38408 Value: 0
Virtual address: 56643 Physical address: 3651 Value: 80
Virtual address: 50887 Physical address: 15047 Value: -79
Virtual address: 49356 Physical address: 38860 Value: 0
Virtual address: 57212 Physical address: 13436 Value: 0
Virtual address: 55174 Physical address: 35718 Value: 53
Virtual address: 26352 Physical address: 4336 Value: 0
Virtual address: 30830 Physical address: 11886 Value: 30
Virtual address: 37682 Physical address: 18994 Value: 36
Virtual address: 20496 Physical address: 23056 Value: 0
Virtual address: 13942 Physical address: 39030 Value: 13
Virtual address: 20418 Physical address: 27586 Value: 19
Virtual address: 45803 Physical address: 39403 Value: -70
Virtual address: 3435 Physical address: 21867 Value: 90
Virtual address: 37008 Physical address: 1936 Value: 0
Virtual address: 5706 Physical address: 39498 Value: 5
Virtual address: 49207 Physical address: 38711 Value: 13
Virtual address: 48964 Physical address: 39748 Value: 0
Virtual address: 57150 Physical address: 13374 Value: 55
Virtual address: 11235 Physical address: 40163 Value: -8
Virtual address: 64866 Physical address: 40290 Value: 63
Virtual address: 31647 Physical address: 40607 Value: -25
Virtual address: 42633 Physical address: 40841 Value: 0
Virtual address: 26669 Physical address: 31533 Value: 0
Virtual address: 37568 Physical address: 33984 Value: 0
Virtual address: 43255 Physical address: 41207 Value: 61
Virtual address: 46429 Physical address: 41309 Value: 0
Virtual address: 12458 Physical address: 41642 Value: 12
Virtual address: 16702 Physical address: 12862 Value: 16
Virtual address: 22106 Physical address: 41818 Value: 21
Virtual address: 31742 Physical address: 40702 Value: 30
Virtual address: 28830 Physical address: 42142 Value: 28
Virtual address: 47512 Physical address: 13720 Value: 0
Virtual address: 56206 Physical address: 42382 Value: 54
Virtual address: 63012 Physical address: 42532 Value: 0
Virtual address: 2967 Physical address: 42903 Value: -27
Virtual address: 22442 Physical address: 22442 Value: 21
Virtual address: 33520 Physical address: 8432 Value: 0
Virtual address: 41328 Physical address: 2928 Value: 0
Virtual address: 56244 Physical address: 42420 Value: 0
Virtual address: 46288 Physical address: 43216 Value: 0
Virtual address: 31239 Physical address: 28679 Value: -127
Virtual address: 35798 Physical address: 16342 Value: 34
Virtual address: 50808 Physical address: 14968 Value: 0
Virtual address: 237 Physical address: 14829 Value: 0
Virtual address: 228 Physical address: 14820 Value: 0
Virtual address: 60113 Physical address: 5585 Value: 0
Virtual address: 22359 Physical address: 22359 Value: -43
Virtual address: 5218 Physical address: 19298 Value: 5
Virtual address: 25228 Physical address: 29324 Value: 0
Virtual address: 49927 Physical address: 36103 Value: -63
Virtual address: 52890 Physical address: 43418 Value: 51
Virtual address: 20556 Physical address: 23116 Value: 0
Virtual address: 25730 Physical address: 25986 Value: 25
Virtual address: 11148 Physical address: 40076 Value: 0
Virtual address: 43553 Physical address: 23841 Value: 0
Virtual address: 57667 Physical address: 22083 Value: 80
Virtual address: 42375 Physical address: 33415 Value: 97
Virtual address: 13499 Physical address: 22715 Value: 46
Virtual address: 35199 Physical address: 1151 Value: 95
Virtual address: 32731 Physical address: 27355 Value: -10
Virtual address: 34348 Physical address: 43564 Value: 0
Virtual address: 1109 Physical address: 6485 Value: 0
Virtual address: 24950 Physical address: 43894 Value: 24
Virtual address: 43 Physical address: 14635 Value: 10
Virtual address: 18957 Physical address: 44045 Value: 0
Virtual address: 34178 Physical address: 44418 Value: 33
Virtual address: 11420 Physical address: 44700 Value: 0
Virtual address: 1507 Physical address: 17891 Value: 120
Virtual address: 46048 Physical address: 18400 Value: 0
Virtual address: 20764 Physical address: 13852 Value: 0
Virtual address: 37373 Physical address: 45053 Value: 0
Virtual address: 23408 Physical address: 45168 Value: 0
Virtual address: 23650 Physical address: 45410 Value: 23
Virtual address: 44584 Physical address: 45608 Value: 0
Virtual address: 26526 Physical address: 45982 Value: 25
Virtual address: 57757 Physical address: 22173 Value: 0
Virtual address: 39542 Physical address: 46198 Value: 38
Virtual address: 46058 Physical address: 18410 Value: 44
Virtual address: 22916 Physical address: 46468 Value: 0
Virtual address: 59969 Physical address: 5441 Value: 0
Virtual address: 36211 Physical address: 20083 Value: 92
Virtual address: 45028 Physical address: 46820 Value: 0
Virtual address: 65476 Physical address: 25540 Value: 0
Virtual address: 41403 Physical address: 3003 Value: 110
Virtual address: 60213 Physical address: 46901 Value: 0
Virtual address: 27216 Physical address: 24656 Value: 0
Virtual address: 43341 Physical address: 2637 Value: 0
Virtual address: 7839 Physical address: 13215 Value: -89
Virtual address: 58529 Physical address: 673 Value: 0
Virtual address: 35358 Physical address: 47134 Value: 34
Virtual address: 52859 Physical address: 43387 Value: -98
Virtual address: 25332 Physical address: 29428 Value: 0
Virtual address: 33629 Physical address: 23389 Value: 0
Virtual address: 16983 Physical address: 47447 Value: -107
Virtual address: 42033 Physical address: 47665 Value: 0
Virtual address: 14874 Physical address: 18458 Value: 14
Virtual address: 2756 Physical address: 8900 Value: 0
Virtual address: 20496 Physical address: 23056 Value: 0
Virtual address: 15854 Physical address: 30446 Value: 15
Virtual address: 55905 Physical address: 21345 Value: 0
Virtual address: 35239 Physical address: 1191 Value: 105
Virtual address: 63124 Physical address: 42644 Value: 0
Virtual address: 13058 Physical address: 24066 Value: 12
Virtual address: 38369 Physical address: 34273 Value: 0
Virtual address: 51661 Physical address: 33229 Value: 0
Virtual address: 55762 Physical address: 23762 Value: 54
Virtual address: 44955 Physical address: 46747 Value: -26
Virtual address: 19703 Physical address: 17655 Value: 61
Virtual address: 40476 Physical address: 47900 Value: 0
Virtual address: 14617 Physical address: 6937 Value: 0
Virtual address: 14541 Physical address: 48333 Value: 0
Virtual address: 10194 Physical address: 31442 Value: 9
Virtual address: 7742 Physical address: 13118 Value: 7
Virtual address: 50663 Physical address: 23015 Value: 121
Virtual address: 538 Physical address: 15386 Value: 0
Virtual address: 41269 Physical address: 2869 Value: 0
Virtual address: 24184 Physical address: 37752 Value: 0
Virtual address: 1432 Physical address: 17816 Value: 0
Virtual address: 26682 Physical address: 31546 Value: 26
Virtual address: 46321 Physical address: 43249 Value: 0
Virtual address: 42449 Physical address: 33489 Value: 0
Virtual address: 41165 Physical address: 48589 Value: 0
Virtual address: 63451 Physical address: 31963 Value: -10
Virtual address: 4409 Physical address: 48697 Value: 0
Virtual address: 41396 Physical address: 2996 Value: 0
Virtual address: 19012 Physical address: 44100 Value: 0
Virtual address: 12948 Physical address: 35476 Value: 0
Virtual address: 45420 Physical address: 49004 Value: 0
Virtual address: 34489 Physical address: 43705 Value: 0
Virtual address: 58291 Physical address: 49331 Value: -20
Virtual address: 1390 Physical address: 17774 Value: 1
Virtual address: 46344 Physical address: 41224 Value: 0
Virtual address: 44121 Physical address: 36697 Value: 0
Virtual address: 43653 Physical address: 23941 Value: 0
Virtual address: 25089 Physical address: 29185 Value: 0
Virtual address: 15276 Physical address: 49580 Value: 0
Virtual address: 22231 Physical address: 41943 Value: -75
Virtual address: 31678 Physical address: 40638 Value: 30
Virtual address: 11061 Physical address: 39989 Value: 0
Virtual address: 14466 Physical address: 48258 Value: 14
Virtual address: 23931 Physical address: 30843 Value: 94
Virtual address: 6901 Physical address: 49909 Value: 0
Virtual address: 20175 Physical address: 28623 Value: -77
Virtual address: 14115 Physical address: 3107 Value: -56
Virtual address: 19388 Physical address: 50108 Value: 0
Virtual address: 62174 Physical address: 50398 Value: 60
Virtual address: 10836 Physical address: 50516 Value: 0
Virtual address: 41814 Physical address: 4438 Value: 40
Virtual address: 64152 Physical address: 33688 Value: 0
Virtual address: 13740 Physical address: 9644 Value: 0
Virtual address: 40925 Physical address: 50909 Value: 0
Virtual address: 25980 Physical address: 51068 Value: 0
Virtual address: 57229 Physical address: 13453 Value: 0
Virtual address: 45733 Physical address: 39333 Value: 0
Virtual address: 55823 Physical address: 21263 Value: -125
Virtual address: 60387 Physical address: 47075 Value: -8
Virtual address: 40803 Physical address: 50787 Value: -40
Virtual address: 40167 Physical address: 51431 Value: 57
Virtual address: 5366 Physical address: 19446 Value: 5
Virtual address: 1152 Physical address: 6528 Value: 0
Virtual address: 56817 Physical address: 3825 Value: 0
Virtual address: 48395 Physical address: 12555 Value: 66
Virtual address: 27299 Physical address: 24739 Value: -88
Virtual address: 14857 Physical address: 18441 Value: 0
Virtual address: 23137 Physical address: 12129 Value: 0
Virtual address: 37444 Physical address: 33860 Value: 0
Virtual address: 10549 Physical address: 51509 Value: 0
Virtual address: 56739 Physical address: 3747 Value: 104
Virtual address: 4277 Physical address: 37045 Value: 0
Virtual address: 63512 Physical address: 38424 Value: 0
Virtual address: 32464 Physical address: 14288 Value: 0
Virtual address: 7543 Physical address: 51831 Value: 93
Virtual address: 37542 Physical address: 33958 Value: 36
Virtual address: 28221 Physical address: 7741 Value: 0
Virtual address: 22095 Physical address: 41807 Value: -109
Virtual address: 49941 Physical address: 36117 Value: 0
Virtual address: 39006 Physical address: 52062 Value: 38
Virtual address: 863 Physical address: 2143 Value: -41
Virtual address: 26726 Physical address: 31590 Value: 26
Virtual address: 41044 Physical address: 48468 Value: 0
Virtual address: 2645 Physical address: 8789 Value: 0
Virtual address: 8972 Physical address: 52236 Value: 0
Virtual address: 17239 Physical address: 52567 Value: -43
Virtual address: 19364 Physical address: 50084 Value: 0
Virtual address: 48391 Physical address: 12551 Value: 65
Virtual address: 21824 Physical address: 32832 Value: 0
Virtual address: 22134 Physical address: 41846 Value: 21
Virtual address: 56368 Physical address: 25136 Value: 0
Virtual address: 41118 Physical address: 48542 Value: 40
Virtual address: 9373 Physical address: 52893 Value: 0
Virtual address: 49918 Physical address: 53246 Value: 48
Virtual address: 63423 Physical address: 31935 Value: -17
Virtual address: 17198 Physical address: 52526 Value: 16
Virtual address: 23095 Physical address: 12087 Value: -115
Virtual address: 23541 Physical address: 45301 Value: 0
Virtual address: 51393 Physical address: 53441 Value: 0
Virtual address: 34904 Physical address: 53592 Value: 0
Virtual address: 31152 Physical address: 34736 Value: 0
Virtual address: 56389 Physical address: 25157 Value: 0
Virtual address: 62874 Physical address: 18842 Value: 61
Virtual address: 23834 Physical address: 30746 Value: 23
Virtual address: 2407 Physical address: 3943 Value: 89
Virtual address: 27118 Physical address: 29166 Value: 26
Virtual address: 35531 Physical address: 47307 Value: -78
Virtual address: 27570 Physical address: 32178 Value: 26
Virtual address: 13545 Physical address: 22761 Value: 0
Virtual address: 1927 Physical address: 27783 Value: -31
Virtual address: 26545 Physical address: 46001 Value: 0
Virtual address: 38435 Physical address: 53795 Value: -120
Virtual address: 10597 Physical address: 51557 Value: 0
Virtual address: 35902 Physical address: 54078 Value: 35
Virtual address: 61972 Physical address: 50196 Value: 0
Virtual address: 24089 Physical address: 37657 Value: 0
Virtual address: 45655 Physical address: 39255 Value: -107
Virtual address: 27991 Physical address: 54359 Value: 85
Virtual address: 53533 Physical address: 34845 Value: 0
Virtual address: 49276 Physical address: 38780 Value: 0
Virtual address: 65321 Physical address: 25385 Value: 0
Virtual address: 49048 Physical address: 39832 Value: 0
Virtual address: 14480 Physical address: 48272 Value: 0
Virtual address: 59538 Physical address: 24978 Value: 58
Virtual address: 26633 Physical address: 31497 Value: 0
Virtual address: 64014 Physical address: 33550 Value: 62
Virtual address: 24132 Physical address: 37700 Value: 0
Virtual address: 29744 Physical address: 1328 Value: 0
Virtual address: 48966 Physical address: 39750 Value: 47
Virtual address: 55060 Physical address: 35604 Value: 0
Virtual address: 32906 Physical address: 18058 Value: 32
Virtual address: 1007 Physical address: 2287 Value: -5
Virtual address: 47546 Physical address: 13754 Value: 46
Virtual address: 23005 Physical address: 46557 Value: 0
Virtual address: 56364 Physical address: 25132 Value: 0
Virtual address: 59221 Physical address: 29781 Value: 0
Virtual address: 29842 Physical address: 1426 Value: 29
Virtual address: 30016 Physical address: 832 Value: 0
Virtual address: 49466 Physical address: 17210 Value: 48
Virtual address: 56450 Physical address: 25218 Value: 55
Virtual address: 36802 Physical address: 6338 Value: 35
Virtual address: 22078 Physical address: 41790 Value: 21
Virtual address: 40292 Physical address: 4964 Value: 0
Virtual address: 57790 Physical address: 22206 Value: 56
Virtual address: 51417 Physical address: 53465 Value: 0
Virtual address: 1524 Physical address: 17908 Value: 0
Virtual address: 2566 Physical address: 8710 Value: 2
Virtual address: 1055 Physical address: 6431 Value: 7
Virtual address: 39881 Physical address: 54729 Value: 0
Virtual address: 53559 Physical address: 34871 Value: 77
Virtual address: 59040 Physical address: 54944 Value: 0
Virtual address: 31997 Physical address: 55293 Value: 0
Virtual address: 36733 Physical address: 6269 Value: 0
Virtual address: 46402 Physical address: 41282 Value: 45
Virtual address: 46774 Physical address: 30646 Value: 45
Virtual address: 41068 Physical address: 48492 Value: 0
Virtual address: 25303 Physical address: 29399 Value: -75
Virtual address: 42140 Physical address: 47772 Value: 0
Virtual address: 10818 Physical address: 50498 Value: 10
Virtual address: 30924 Physical address: 11980 Value: 0
Virtual address: 63933 Physical address: 55485 Value: 0
Virtual address: 21005 Physical address: 55565 Value: 0
Virtual address: 32369 Physical address: 14193 Value: 0
Virtual address: 52916 Physical address: 43444 Value: 0
Virtual address: 63207 Physical address: 42727 Value: -71
Virtual address: 28612 Physical address: 56004 Value: 0
Virtual address: 1934 Physical address: 27790 Value: 1
Virtual address: 39577 Physical address: 46233 Value: 0
Virtual address: 44227 Physical address: 36803 Value: 48
Virtual address: 25342 Physical address: 29438 Value: 24
Virtual address: 17622 Physical address: 8150 Value: 17
Virtual address: 20234 Physical address: 27402 Value: 19
Virtual address: 8961 Physical address: 52225 Value: 0
Virtual address: 56686 Physical address: 3694 Value: 55
Virtual address: 40834 Physical address: 50818 Value: 39
Virtual address: 44721 Physical address: 45745 Value: 0
Virtual address: 43310 Physical address: 2606 Value: 42
Virtual address: 1604 Physical address: 56132 Value: 0
Number of Translated Addresses = 500
Page Faults = 220
Page Fault Rate = 0.440
TLB Hits = 35
TLB Hit Rate = 0.070
